openqa-investigate

```diff
@@ -66,9 +66,8 @@ clone() {
 
     # output: { "$id": $clone_id }
     clone_id=$(echo "$out" | runjq -r ".\"$id\"")
-    url="$scheme://$host/t$clone_id"
     # Create markdown list entry
-    echo "* *$name*: $url"
+    echo "* *$name*: t#$clone_id"
```

Comment:
The job status bubbles created via javascript when loading comments don't recognize this syntax, e.g. compare
https://openqa.opensuse.org/tests/3734688#comments (good) vs.
https://openqa.opensuse.org/tests/3751441#comments (bad)